src/style.css

```
@@ -0,0 +1,81 @@
+body {
+  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+  align-items: stretch;
+}
+
+.block {
+  width: 33%;
+  height: 300px;
+  color: white;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-grow: 1;
+}
+
+.block:nth-child(1) {
+  background-color: rgb(255, 0, 0);
+}
+
+.block:nth-child(2) {
+  background-color: rgb(204, 0, 0);
+}
+
+.block:nth-child(3) {
+  background-color: rgb(153, 0, 0);
+}
+
+.block:nth-child(4) {
+  background-color: rgb(102, 0, 0);
+}
+
+.block:nth-child(5) {
+  background-color: rgb(51, 0, 0);
+}
+
+.block:nth-child(6) {
+  background-color: rgb(0, 0, 0);
+}
+
+@media (max-width: 1200px) {
+  .block:nth-child(6) {
+    order: 4;
+  }
+  .block:nth-child(5) {
+    order: 5;
+  }
+  .block:nth-child(4) {
+    order: 6;
+  }
+}
+@media (max-width: 750px) {
```

```suggestion
@media (max-width: 750px) {
```
Чому такі цифри 750 та 450?
В умові сказано, що блоки мають бути не вужчі 300 пікселів. Отже, потрібно відштовхуватись від брейкпойнтів 300, 600, 900